src/components/carousel/carousel.js

```diff
@@ -87,6 +111,8 @@ Carousel.propTypes = {
   autoplaySpeed: PropTypes.number,
   /** Whether or not to display pagination */
   hasPaging: PropTypes.bool,
+  /** Slider to handle the pagination 'top' or 'bottom' */
+  primarySlider: PropTypes.string,
```

Comment:
Maybe we can set here more specific prop type, like:
`primarySlider: PropTypes.oneOf(['top', 'bottom']),`

Reply:
Indeed. Thanks!!